lib/const.rb

```diff
@@ -1,7 +1,7 @@
 # lib/const.rb
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.45.2'
+CREW_VERSION = '1.45.3'
```

Comment:
Do we need to bump `CREW_VERSION` for only an addition to the rename list in `lib/fixup.rb`?

Reply:
I think it makes sense to bump the version for ANY changes, actually.  This makes it possible to go back and find where changes were made.

Reply:
The exception would be packages updates, of course.

Reply:
I figured it was safer to do so?